src/Files/ServicesImplementation/SettingsServices/FileTagsSettingsService.cs

```diff
@@ -1,24 +1,26 @@
-﻿using Files.Filesystem;
-using Files.Models.JsonSettings;
+﻿using Files.Backend.Models;
+using Files.Models;
+using Files.Backend.Models.JsonSettings;
+using Files.Backend.Services.Settings;
 using Microsoft.Toolkit.Uwp;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using Windows.Storage;
 
-namespace Files.Services.Implementation
+namespace Files.ServicesImplementation.SettingsServices
 {
     public sealed class FileTagsSettingsService : BaseJsonSettingsModel, IFileTagsSettingsService
     {
         public event EventHandler OnSettingImportedEvent;
 
-        private static readonly List<FileTag> s_defaultFileTags = new List<FileTag>()
+        private static readonly List<IFileTag> s_defaultFileTags = new List<IFileTag>()
         {
-            new FileTag("Blue", "#0072BD"),
-            new FileTag("Orange", "#D95319"),
-            new FileTag("Yellow", "#EDB120"),
-            new FileTag("Green", "#77AC30"),
-            new FileTag("Azure", "#4DBEEE")
+            new FileTagModel("Blue", "#0072BD"),
```

Comment:
@d2dyno1 Is FileTagModel a good name? Maybe we can drop the "Model" part 🤔

Reply:
I don't see it being a problem, we should keep it as is ig